test/files/run/repl-implicits-nopredef.scala

```diff
@@ -0,0 +1,10 @@
+import scala.tools.partest.ReplTest
+import scala.tools.nsc.Settings
+
+object Test extends ReplTest {
+  override def transformSettings(settings: Settings): Settings = {
+    settings.nopredef.value = true
+    settings
+  }  
+  def code = ":implicits\n"
```

Comment:
The newline should be unnecessary. ILoop.runForTranscript adds one for good measure after stripping yours: `val input = new BufferedReader(new StringReader(code.trim + "\n"))`.